test/unit/dependency_checker_test.rb

```diff
@@ -29,78 +30,34 @@ def listed?(_references, violation_type:)
     test "recognizes simple cross package reference" do
       source_package = Package.new(name: "components/sales", config: { "enforce_dependencies" => true })
       checker = dependency_checker
-
-      reference =
-        Reference.new(
-          source_package,
-          "some/path.rb",
-          ConstantDiscovery::ConstantContext.new(
-            "::SomeName",
-            "some/location.rb",
-            @destination_package,
```

Comment:
Now `@destination_package` is not needed in this test anymore.

Reply:
Wow, impressive spotting! I've removed the `@destination_package` ivar and things seem fine 👍